proposals/lip-0063.md

```diff
@@ -537,6 +577,35 @@ def createGenesisDataObj():
     return genesisDataObj
 ```
 
+###### Assets Entry for Interoperability Module
+
+Let `genesisInteroperabilityStoreSchema` be as defined in [LIP 0045](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md#genesis-state-initialization). The `assets` entry for the Interoperability module is added by the logic defined in the function `addInteropModuleEntry` in the following pseudo code. The function `getMainchainID` returns the chain ID of the mainchain in the current network, it is defined in [LIP 0037](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#mainchain-chain-id).
+
+```python
+def addInteropModuleEntry():
+    InteropObj = {}
+    InteropObj.outboxRootSubstore = []
+    InteropObj.chainDataSubstore = []
+    InteropObj.channelDataSubstore = []
+    InteropObj.chainValidatorsSubstore = []
+    ownChainAccount = {
+        "name": CHAIN_NAME_MAINCHAIN,
+        "chainId": getMainchainID(),
```

Comment:
I would recommend to merge the main branch into your branch. The constants table there already defines `CHAIN_ID_MAINCHAIN`. Then you can use this value here.

Reply:
I would prefer to leave the function explicitly, since we also want to migrate in Testnet. There is a reference to LIP 0037 after the constants table, but I think it is nice to be more precise and mention it directly in the pseudocode.
Also I have confirmed with the Platform team that this function is available.

Reply:
I would still strongly favor using `CHAIN_ID_MAINCHAIN`. We define the computation of the snapshot block in the context of Lisk Core 3.0 where `OWN_CHAIN_ID` should not be defined and therefore the function `getMainchainID()` does not really make sense. In some place of the Migrator, we will need to hardcode `CHAIN_ID_MAINCHAIN` as this value cannot be computed. And, as already mentioned below the constants table, this hard-coded value will need to be different in depending on where we do the migration. Therefore, I would find it better to stick with the notation `CHAIN_ID_MAINCHAIN` in the whole LIP (even if internally in the migrator tool they introduce a function `getMainchainID()` for it).

Reply:
Looks all good.